test/abap/pretty_printer.ts

```diff
@@ -3,6 +3,17 @@ import {PrettyPrinter} from "../../src/abap/pretty_printer";
 import {MemoryFile} from "../../src/files";
 import {Registry} from "../../src/registry";
 
+const testTitle = (text: string): string => { return text.split("\n")[0]; };
+const createRunner = (test: any, methodName: string): (() => void) => {
+  return () => {
+    const reg = new Registry().addFile(new MemoryFile("zfoo.prog.abap", test.input)).parse();
+    expect(reg.getABAPFiles().length).to.equal(1);
+    const printer = new PrettyPrinter(reg.getABAPFiles()[0], test.options);
+    const result = (printer as any)[methodName]();
```

Comment:
this looks fishy, it breaks all assisted refactoring and where-used

Reply:
removed